README.md

```diff
@@ -50,4 +51,28 @@ enum Level {
 
 Level myVar = Level.MEDIUM    
 System.out.println(myVar)    // returns MEDIUM
-```
\ No newline at end of file
+```
+
+***Reserved Words***
+![list of reserved words](img.png)
+
+***Variable Types***
+
+**Primitive** - There are 8 types that all represent the actual value of the variable:
```

Comment:
primitives have their Object wrapper classes, some of them mentioned below with examples